Client/Cards/index.ts

```diff
@@ -30,6 +30,10 @@ export class Cards extends List<
 		const result = await this.connection.post<model.CardResponseV2>("v2/cards/virtual", request)
 		return model.ErrorResponse.is(result) ? result : this.map(result)
 	}
+	async createWithMultipart(formData: FormData) {
+		const result = await this.connection.postMultipart<model.CardResponseV2>("v2/cards/virtual", formData)
+		return model.ErrorResponse.is(result) ? result : this.map(result)
+	}
```

Comment:
this probably isnt a very good way of doing it, you make the calling code do most of the work

why not have it `createCardWithRemittanceAdvice(CreateCardRequest, File)` and build the request in here